TFP-5185: Fase 1 hiver ut delvis godkjent arbeid til manuell behandling for å unngå overstyringer

sirimykland opened the pull request.

sirimykland committed:
TFP-5185: Fase 1 hiver ut delvis godkjent arbeid til manuell behandling for å unngå overstyringer

palfi commented:
Vi må huske på å sjekke at vi ikke spør om mors stillingsprosent ved utsettelser når det bfhr. Regner med at vi ikke skal trenge mors stilling i disse periodene?

espenwaaga committed:
coauthor
espenwaaga <36693523+espenwaaga@users.noreply.github.com>
palfi <1370608+palfi@users.noreply.github.com>
AnjaAalerud <52454508+AnjaAalerud@users.noreply.github.com>

espenwaaga committed:
coauthors
Co-authored-by: espenwaaga <36693523+espenwaaga@users.noreply.github.com>
Co-authored-by: palfi <1370608+palfi@users.noreply.github.com>
Co-authored-by: AnjaAalerud <52454508+AnjaAalerud@users.noreply.github.com>
